mail_gateway_telegram/models/mail_gateway_telegram.py

```diff
@@ -0,0 +1,387 @@
+# Copyright 2018 ACSONE SA/NV
```

Comment:
?

Reply:
Copy paste when we create files is our worst friend...